content/codes.md

```diff
@@ -65,7 +65,9 @@ toc: true
   [youtube](https://www.youtube.com/watch?v=zZeUvHkOOAM&list=PLD4D607C2FA317E6D&index=147) |
   [notes in Chinese](https://blog.seisman.info/conversion-of-different-sac-formats):
   The most commonly used seismic data processing and plotting software
-- [CubeTools](https://digos.eu/downloads-docs/): Data Format conversion software for CUBE data
+- [CubeTools](https://digos.eu/downloads-docs/) |
```

Comment:
It's rarely used, no?

Reply:
Not really. Temporal seismic stations might use this format.

Reply:
As I understand it, data-cube is a data format for a specific type of seismometer. People who use this type of seismometer should already know this software, right?

We already have too many links on this site. I don't want to add links that are only useful to very few people.

Reply:
Yes, but this format is not really rarely used. As you could see, `uafgeotools` is a seismological group in Alaska, and they have some python scripts for this tool (`cube_conversion`). GEOFON also uses this data format. A group in Singapore also has this kind of seismometer.

Although people who use this type of seismometer might already know this software, but it still takes a lot of time to learn it based on the offical manual. I find this `cube_conversion` python script is useful. So, we might want to keep it?